src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -615,6 +619,8 @@ private static void setViolations(TestInputConfiguration.Builder inputConfigBuil
                 VIOLATIONS_SOME_LINES_ABOVE_PATTERN.matcher(lines.get(lineNo));
         final Matcher violationsSomeLinesBelowMatcher =
                 VIOLATIONS_SOME_LINES_BELOW_PATTERN.matcher(lines.get(lineNo));
+        final Matcher violationsDefault =
+                VIOLATION_DEFAULT.matcher(lines.get(lineNo));
```

Comment:
add usage of it before `else if (useFilteredViolations) {`

something like:
```
else if (violationsDefault.matches()) {
            final int violationLineNum = lineNo + 1;
            inputConfigBuilder.addViolation(violationLineNum, null);
        }
else if (useFilteredViolations) {
```

Reply:
Done